packages/bibliography/src/main.rs

```diff
@@ -286,38 +290,204 @@ fn transform_bibliography(to: &str, input: &Value) {
+                    let mut styled_prefix = vec![];
+
+                    if let Some(mut prefix) = prefix {
+                        using_prefix = true;
+                        styled_prefix.append(&mut prefix);
+                    }
+
+                    item.push(raw!(format!(
+                        r"\leavevmode \hypertarget{{bibentry:{}}}{{",
+                        entry.entry.key()
+                    )));
+                    item.append(&mut display(&entry.display));
+                    item.push(raw!("}".to_string()));
+                    item.push(module!("newline", ""));
+                    item.push(raw!("\n"));
+
+                    (styled_prefix, item)
+                })
+                .collect();
+
+            // If the we have nothing to show, return early with an empty vec
+            if bibitems.is_empty() {
+                return vec![];
             }
-            if let Some(prefix) = &entry.prefix {
-                // If we have a prefix (which is the number in [] for IEEE), encase it in brackets
-                output.push(text!("["));
-                output.append(&mut display(prefix));
-                output.push(text!("] "));
+
+            let mut output = Vec::new();
+            output.push(raw!("\n"));
+            output.push(import!("\\usepackage{tabularx}"));
+
+            // If we have a prefix, use two columns otherwise one
+            if using_prefix {
+                output.push(raw!(
+                    r"\renewcommand{\arraystretch}{1.5}
+\begin{tabularx}{\textwidth}{p{0.3cm} >{\raggedright\arraybackslash} X}
+"
+                ));
+            } else {
+                output.push(raw!(
+                    r"\renewcommand{\arraystretch}{1.5}
+\begin{tabularx}{\textwidth}{>{\raggedright\arraybackslash} X}
+"
+                ));
             }
-            if to == "latex" {
-                output.push(raw!(format!(
-                    r"\hypertarget{{bibentry:{}}}{{",
-                    entry.entry.key()
-                )));
-            } else if to == "html" {
-                output.push(raw!(format!(
-                    r#"<span id="bibentry:{}">"#,
-                    entry.entry.key()
-                )));
+
+            output.extend(
+                bibitems
+                    .into_iter()
+                    .map(|(mut prefix, mut item)| {
+                        let mut result = vec![];
+                        if using_prefix {
+                            result.append(&mut prefix);
+                            result.push(raw!("&"));
+                        }
+                        result.append(&mut item);
+                        result
+                    })
+                    .flatten(),
+            );
+
+            output.push(raw!(
+                r"\end{tabularx}
+\renewcommand{\arraystretch}{1}
+"
+            ));
+            output
+        }
+        // For html output we use a simple css grid layout
```

Comment:
It looks like bibliographies without prefix use `display: flex` now, could update comment to reflect that

Reply:
thanks! Will fix!